lbrynet/lbrynet_daemon/LBRYDaemon.py

```diff
@@ -53,8 +54,6 @@
 from lbrynet.lbryfile.LBRYFileMetadataManager import DBLBRYFileMetadataManager, TempLBRYFileMetadataManager
 # from lbryum import LOG_PATH as lbryum_log
 
-log = logging.getLogger(__name__)
```

Comment:
@jobevers are you sure this doesnt belong? is `log` magically defined everywhere now?